src/state-summary/state-card-select.ts

```diff
@@ -0,0 +1,99 @@
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu-light";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+import {
+  css,
+  CSSResultGroup,
+  html,
+  LitElement,
+  PropertyValues,
+  TemplateResult,
+} from "lit";
+import { customElement, property } from "lit/decorators";
+import { stopPropagation } from "../common/dom/stop_propagation";
+import { computeStateName } from "../common/entity/compute_state_name";
+import "../components/entity/state-badge";
+import { SelectEntity, setSelectOption } from "../data/select";
+import type { HomeAssistant } from "../types";
+
+@customElement("state-card-select")
+class StateCardSelect extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public stateObj!: SelectEntity;
+
+  protected render(): TemplateResult {
+    return html`
+      <state-badge .stateObj=${this.stateObj}></state-badge>
+      <paper-dropdown-menu-light
+        .label=${computeStateName(this.stateObj)}
+        @iron-select=${this._selectedOptionChanged}
+        @click=${stopPropagation}
+      >
+        <paper-listbox slot="dropdown-content">
+          ${this.stateObj.attributes.options.map(
+            (option) =>
+              html`
+                <paper-item .option=${option}
+                  >${(this.stateObj.attributes.device_class &&
+                    this.hass.localize(
+                      `component.select.state.${this.stateObj.attributes.device_class}.${option}`
+                    )) ||
+                  this.hass.localize(`component.select.state._.${option}`) ||
+                  option}</paper-item
+                >
+              `
+          )}
+        </paper-listbox>
+      </paper-dropdown-menu-light>
+    `;
+  }
+
+  protected updated(changedProps: PropertyValues) {
+    super.updated(changedProps);
+    if (!changedProps.has("stateObj")) {
+      return;
+    }
+    // Update selected after rendering the items or else it won't work in Firefox
+    this.shadowRoot!.querySelector(
+      "paper-listbox"
+    )!.selected = this.stateObj.attributes.options.indexOf(this.stateObj.state);
+  }
+
+  private async _selectedOptionChanged(ev) {
```

Comment:
And if we are not going to await it anyway, this doesn't need to be async... But we can also await it, like balloon said, it does make stack traces better readable... It's micro 🤷‍♂️ 
```suggestion
  private _selectedOptionChanged(ev) {
```